project_scrum/views/project_project.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="project_ps_view" model="ir.ui.view">
+        <field name="name">project.project.tree</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="use_scrum" string="Using Scrum" readonly="1" />
+            </field>
+            <field name="user_id" position="attributes">
+                <attribute name="string">Scrum Master/Project
+                    Manager
+                </attribute>
+            </field>
+        </field>
+    </record>
+
+    <record id="edit_project" model="ir.ui.view">
+        <field name="name">Edit Project</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='options_active']" position="inside">
+                <div>
+                    <field name="use_scrum" class="oe_inline" />
+                    <label for="use_scrum" string="Use Scrum" />
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="act_project_tasks" model="ir.actions.act_window">
```

Comment:
XML shortcut tags <act_window> are no longer valid, and must be replaced by its full definition through <record> tag.